code/game/gamemodes/objective.dm

```diff
@@ -439,6 +461,17 @@ GLOBAL_LIST_EMPTY(objectives)
 			return FALSE
 	return TRUE
 
+/datum/objective/captured
+	name = "captured"
+	explanation_text = "Escape on the shuttle while being in custody."
+
+/datum/objective/captured/check_completion()
+	var/list/datum/mind/owners = get_owners()
+	for(var/datum/mind/M in owners)
+		if(considered_escaped(M) || !considered_alive(M))
+			return FALSE
+	return TRUE
+
```

Comment:
An "end the shift with mindshield/tracking implant" would make more sense, considering shitcurity sometimes implants antags with mindshield when paroled and the otherwise struggle of trying to get a mindshield/tracking.